### PR TITLE
Add new PocketBook devices PB-Basic Lux 4 (PB618)

### PR DESCRIPTION
**Pull request by ElimGarak1**
I now also own a PB-Basic Lux 4 (PB618) and KOReader runs excellently on it.
[crash.log](https://github.com/koreader/koreader/files/12840018/crash.log)

IMG_20231008_090913_427

IMG_20231008_090543_351

IMG_20231008_090623_582

IMG_20231008_090654_619

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/10987)

### CODE REVIEW

File: frontend/device/pocketbook/device.lua [hunk 1]
```diff
@@ -506,6 +506,16 @@ local PocketBook617 = PocketBook:extend{
...
 }
 
+-- PocketBook Basic Lux 4 (618)
+local PocketBook618 = PocketBook:extend{
+    model = "PB618",
+    display_dpi = 212,
+    isTouchDevice = yes,
+    hasDPad = no,
+    hasFewKeys = yes,
+    hasFrontlight = yes,
```
Comment (rjd22) [inline frontend/device/pocketbook/device.lua:516]: Just checking but the device does not have support for natural light like the previous touch lux?

File: frontend/device/pocketbook/device.lua [hunk 2]
```diff
@@ -506,6 +506,16 @@ local PocketBook617 = PocketBook:extend{
     hasNaturalLight = yes,
 }
 
+-- PocketBook Basic Lux 4 (618)
+local PocketBook618 = PocketBook:extend{
+    model = "PB618",
```
Comment (rjd22) [inline frontend/device/pocketbook/device.lua:511]: @Frenzie is there a preference here on how to name the model id?

For the lux 3 this was `PBBLux3`. I can imagine that makes it easier to figure out what model were talking about without cross referencing.

File: frontend/device/pocketbook/device.lua (hunk 1 above)
Comment (ElimGarak1) [reply frontend/device/pocketbook/device.lua:516]: > Just checking but the device does not have support for natural light like the previous touch lux?

Only white LEDs are installed in the PB618, PB617 has white and orange LEDs.

IMG_20231008_101741_476

PB618 has a touchscreen, but PB617 does not. That's why PB617 has a D-pad, but PB618 doesn't.

**Review by rjd22 — APPROVED** [review]
LGTM

File: frontend/device/pocketbook/device.lua (hunk 2 above)
Comment (Frenzie) [reply frontend/device/pocketbook/device.lua]: Yes, something like PBLux4, see #4479.

File: frontend/device/pocketbook/device.lua [hunk 3]
```diff
@@ -506,6 +506,16 @@ local PocketBook617 = PocketBook:extend{
     hasNaturalLight = yes,
 }
 
+-- PocketBook Basic Lux 4 (618)
+local PocketBook618 = PocketBook:extend{
+    model = "PBBLux4",
+    display_dpi = 212,
+    isTouchDevice = yes,
+    hasDPad = no,
+    hasFewKeys = yes,
```
Comment (Frenzie) [inline frontend/device/pocketbook/device.lua:515]:
```suggestion
```

**Comment by Frenzie** [comment]
Misclick I assume? ;-)

**Review by Frenzie — APPROVED** [review]

**Pull request merged**